components/masking_functions/include/masking_functions/string_service_tuple.hpp

```diff
@@ -0,0 +1,42 @@
+/* Copyright (c) 2018, 2019 Francisco Miguel Biete Banon. All rights reserved.
+   Copyright (c) 2023 Percona LLC and/or its affiliates. All rights reserved.
+
+   This program is free software; you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation; version 2 of the License.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program; if not, write to the Free Software Foundation,
+   51 Franklin Street, Suite 500, Boston, MA 02110-1335 USA */
+
+#ifndef MASKING_FUNCTIONS_STRING_SERVICE_TUPLE_HPP
+#define MASKING_FUNCTIONS_STRING_SERVICE_TUPLE_HPP
+
+#include <mysql/components/service.h>
+#include <mysql/components/services/mysql_string.h>
+
+#include "masking_functions/string_service_tuple_fwd.hpp"
+
+namespace masking_functions {
+
+struct string_service_tuple {
```

Comment:
As I have said above, I think it is worth to add comment  describing what principle was used to group these services together. E.g. by saying somehting like: Tuple with string and charset-related service handles.

Reply:
Added a description.

Reply:
Thanks!